jobbergate-api/jobbergate_api/apps/job_scripts/routers.py

```diff
@@ -115,7 +116,7 @@ async def job_script_create(
     logger.debug(f"Job-script created: {dict(job_script_data)}")
 
     response = JobScriptResponse(
-        **job_script_data,
+        **{**job_script_data, "application_name": application.application_name},
```

Comment:
If you agree, we can achieve the same effect with a better readability:

```suggestion
        **job_script_data,
        application_name=application.application_name,
```

Reply:
Actually, this won't work if `job_script_data` already includes `application_name`. Python will complain about duplicate keys.

Reply:
I see your point. @matheushent, feel free to ignore both of my code suggestions in this PR.

Reply:
> I see your point. @matheushent, feel free to ignore both of my code suggestions in this PR.

well, I implemented your suggestions and the tests passed.

Reply:
Yes, I've checked there, `job_script_data` is not expected to include `application_name`.